M2/Macaulay2/packages/Varieties/doc-maps.m2

```diff
@@ -0,0 +1,243 @@
+undocumented {
+    sheafMap, -- deprecated
+    -- TODO: document some of these
+    (symbol SPACE, SheafMap, ZZ),
+    (symbol +,   SheafMap, SheafMap),
+    (symbol *,   SheafMap, SheafMap),
+    (symbol *,   RingElement, SheafMap), (symbol *, ZZ, SheafMap),
+    (symbol -,   SheafMap, SheafMap), (symbol -, SheafMap),
+    (symbol **,  SheafMap, SheafMap),
+    (symbol **,  SheafMap, SheafOfRings),  (symbol **, SheafOfRings, SheafMap),
+    (symbol **,  SheafMap, CoherentSheaf), (symbol **, CoherentSheaf, SheafMap),
+    (symbol |,   SheafMap, SheafMap),
+    (symbol ||,  SheafMap, SheafMap),
+    (symbol ==,  SheafMap, SheafMap),
+    (symbol ==,  SheafMap, ZZ), (symbol ==, ZZ, SheafMap),
+    }
```

Comment:
Why are these undocumented?

Reply:
Mainly because nobody got around to it, but if people are already familiar with the same operation with matrices, is there a point in repeating them? I think having a tutorial that covers a bunch of things like this is more useful than documenting each small part of the parlance.